extensions/pyRevitTools.extension/pyRevit.tab/Drawing Set.panel/Print Sheets.pushbutton/script.py

```diff
@@ -1397,6 +1397,7 @@ def cleanup_sheetnumbers(doc):
 
 # verify model is printable
 forms.check_modeldoc(exitscript=True)
+selection = revit.uidoc.Selection.SetElementIds(List[DB.ElementId]([revit.DB.ElementId(2147483647)]))
```

Comment:
@alexdaversa I'm going to change this to use `DB.ElementId.InvalidElementId` and will add this to `pyrevit.revit.selection` as helper method.

Reply:
https://github.com/eirannejad/pyRevit/commit/704f39fb8437123829b7c0d798c55bfd80c1bef6